docs/source/filingtypes.rst

```diff
@@ -0,0 +1,62 @@
+.. _filingtypes:
+
+Filing Types
+============
+
+The SECEdgar package provides a ``Filing`` class which acts as an API to SEC EDGAR document filings.
```

Comment:
We can add a URL link for `SEC EDGAR document filings`. Maybe this one `https://www.sec.gov/edgar/searchedgar/companysearch.html`